packages/lib-classifier/src/store/annotations/DrawingToolAnnotations/PointAnnotation.spec.js

```diff
@@ -0,0 +1,10 @@
+import { types } from 'mobx-state-tree'
+
+const PointAnnotation = types.model('PointAnnotation', {
+  tool: types.number,
+  toolType: types.string,
+  x: types.number,
+  y: types.number
+})
+
+export default PointAnnotation
```

Comment:
This should be a spec for the point annotation model, not a copy of the model definition.

Reply:
🤦‍♂